doc/api/embedding.md

```diff
@@ -167,8 +167,1327 @@ int RunNodeInstance(MultiIsolatePlatform* platform,
+  current thread while waiting for a new task to arrive.
+- **JavaScript/Native interop APIs.** They rely on the existing [node-api][]
+  set of functions. The embedding APIs provide access to functions that
+  retrieve or create `napi_env` instances related to a runtime instance.
+
+## API reference
+
+The C embedder API is split up by the four major groups described above.
+
+### Global platform APIs
+
+#### Data types
+
+##### `node_embedding_platform`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+This is an opaque pointer that represents a Node.js platform instance.
+Node.js allows only a single platform instance per process.
+
+##### `node_embedding_platform_config`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+This is an opaque pointer that represents a Node.js platform configuration
+instance.
+
+##### `node_embedding_status`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+The status code returned from the C Node.js embedding APIs.
+
+```c
+typedef enum {
+  node_embedding_status_ok = 0,
+  node_embedding_status_generic_error = 1,
+  node_embedding_status_null_arg = 2,
+  node_embedding_status_bad_arg = 3,
+  node_embedding_status_out_of_memory = 4,
+  node_embedding_status_error_exit_code = 512,
+} node_embedding_status;
+```
+
+- `node_embedding_status_ok` - No issues.
+- `node_embedding_status_generic_error` - Generic error code.
+- `node_embedding_status_null_arg` - One of non-optional arguments passed as
+  NULL value.
+- `node_embedding_status_bad_arg` - One of the arguments has wrong value.
+- `node_embedding_status_out_of_memory` - Failed to allocate memory.
+- `node_embedding_status_error_exit_code` - A bit flag added to the Node.js exit
+   code value if the error status is associated with an error code.
+
+##### `node_embedding_platform_flags`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Flags are used to initialize a Node.js platform instance.
+
+```c
+typedef enum {
+  node_embedding_platform_flags_none = 0,
+  node_embedding_platform_flags_enable_stdio_inheritance = 1 << 0,
+  node_embedding_platform_flags_disable_node_options_env = 1 << 1,
+  node_embedding_platform_flags_disable_cli_options = 1 << 2,
+  node_embedding_platform_flags_no_icu = 1 << 3,
+  node_embedding_platform_flags_no_stdio_initialization = 1 << 4,
+  node_embedding_platform_flags_no_default_signal_handling = 1 << 5,
+  node_embedding_platform_flags_no_init_openssl = 1 << 8,
+  node_embedding_platform_flags_no_parse_global_debug_variables = 1 << 9,
+  node_embedding_platform_flags_no_adjust_resource_limits = 1 << 10,
+  node_embedding_platform_flags_no_use_large_pages = 1 << 11,
+  node_embedding_platform_flags_no_print_help_or_version_output = 1 << 12,
+  node_embedding_platform_flags_generate_predictable_snapshot = 1 << 14,
+} node_embedding_platform_flags;
+```
+
+These flags match to the C++ `node::ProcessInitializationFlags` and control the
```

Comment:
Does it make sense to have a link to the definition of the node::ProcessInitializationFlags

Reply:
Good point! I will do that.